www/locale/fi/LC_MESSAGES/django.po

```diff
@@ -540,150 +538,182 @@ msgstr ""
-msgid "Nickname"
-msgstr "Kutsumanimi"
-
-#: www/templates/www/usersettings.html:31
-msgid "Matrix ID"
-msgstr "Matrix ID"
-
-#: www/templates/www/usersettings.html:35
-msgid "NFC Card"
-msgstr "NFC-kortti"
-
-#: www/templates/www/usersettings.html:39
-msgid "Release"
-msgstr "Vapauta"
-
-#: www/templates/www/usersettings.html:43
-msgid "Claim NFC card"
-msgstr "Käytä NFC-korttia"
-
-#: www/templates/www/usersettings.html:46
-msgid ""
-"No NFC card claimed - go to door, use a card and claim it here in 5 minutes"
-msgstr ""
-"Ei NFC-korttia yhdistettynä - mene ovelle, leimaa kortti ja yhdistä se "
-"täällä viiden minuutin kuluessa"
-
-#: www/templates/www/usersettings.html:55
-msgid "Save changes"
-msgstr "Tallenna muutokset"
+#: www/templates/www/usersettings.html:23
+msgid "Save"
+msgstr "Tallenna"
 
-#: www/templates/www/usersettings.html:59
+#: www/templates/www/usersettings.html:30
 msgid "Extra Services"
 msgstr "Lisäpalvelut"
 
-#: www/templates/www/usersettings.html:61
+#: www/templates/www/usersettings.html:31
 msgid ""
 "Here you can see extra services that you can subscribe yourself. You can "
 "only unsubscribe services when you have\n"
 "  paid them fully. "
 msgstr ""
-"Tässä näet lisäpalvelut jotka voit halutessasi ottaa käyttöön. "
-"Voit poistaa käytöstä palveluita ainoastaan kun ne ovat maksettu ja "
-"aktiivisessa tilassa."
+"Tässä näet lisäpalvelut jotka voit halutessasi ottaa käyttöön. Voit poistaa "
+"käytöstä palveluita ainoastaan kun ne ovat maksettu ja aktiivisessa tilassa."
+
+#: www/templates/www/usersettings.html:38
+msgid "Information"
+msgstr "Tietoja"
 
-#: www/templates/www/usersettings.html:66
+#: www/templates/www/usersettings.html:53
 msgid "Subscribe service"
 msgstr "Ota käyttöön"
 
-#: www/templates/www/usersettings.html:73
+#: www/templates/www/usersettings.html:70
 msgid "Unsubscribe service"
 msgstr "Poista käytöstä"
 
-#: www/views.py:179
+#: www/templates/www/usersettings.html:81
+msgid "No services available to subscribe or unsubscribe "
+msgstr "Ei käyttöönotettavia tai poistettavia palveluita"
+
+#: www/templates/www/usersettings.html:87
+msgid "NFC Card access"
+msgstr "NFC-Kortti sisäänpääsy"
+
+#: www/templates/www/usersettings.html:91
+msgid "NFC Card"
+msgstr "NFC-Kortti"
+
+#: www/templates/www/usersettings.html:105
+msgid "Release"
+msgstr "Vapauta"
+
+#: www/templates/www/usersettings.html:121
+msgid "capture"
+msgstr "haltuunota"
+
+#: www/templates/www/usersettings.html:128
+msgid ""
+"No recent unclaimed NFC cards. Go to door, use a card and claim it here in 5 "
+"minutes"
+msgstr ""
+"Ei NFC-korttia yhdistettynä - mene ovelle, leimaa kortti ja yhdistä se "
+"täällä 5 minuutin kuluessa"
+
+#: www/templates/www/usersettings.html:139
+msgid ""
+"If you have questions and or want to change something that is not available "
+"here please send an email."
+msgstr ""
+"Jos sinulla on kysyttävää tai haluat muuttaa jotain mitä ei ole saatavilla "
+"täällä voit lähettää ehdotuksen sähköpostilla."
```

Comment:
taitaa olla englanninkielisessäkin pilkku virhe. pitäisi varmaan olla "something that is not available here, please send an email"

ja suomennos
```
"Jos sinulla on kysyttävää tai haluat muuttaa jotain mitä ei ole saatavilla "
"lähetä pyyntösi sähköpostilla."
```